README.md

```diff
@@ -1,29 +1,17 @@
 # jenkins-docker-master
 
 [![license](https://img.shields.io/github/license/mashape/apistatus.svg)](LICENSE)
+[![release](https://img.shields.io/github/release/target/jenkins-docker-master.svg)](https://github.com/target/jenkins-docker-master/releases/latest)
+[![docker](https://img.shields.io/docker/automated/target/jenkins-docker-master.svg)](https://hub.docker.com/r/target/jenkins-docker-master)
 
-## Supported tags and respective `Dockerfile` links
-
-- [`latest`, `2.73.3-1` (*Dockerfile*)](https://github.com/target/jenkins-docker-master/blob/master/Dockerfile)
-- [`debug-latest`, `debug-2.73.3-1` (*Dockerfile.debug*)](https://github.com/target/jenkins-docker-master/blob/master/Dockerfile.debug)
-
-## Quick reference
-
-- **Where to get help**:
-  [The Jenkins users google group](https://groups.google.com/forum/?nomobile=true#!forum/jenkinsci-users) or [Stack Overflow](https://stackoverflow.com/search?tab=newest&q=jenkins)
-
-- **Where to file issues**:
-  [https://github.com/target/jenkins-docker-master/issues](https://github.com/target/jenkins-docker-master/issues)
-
-- **Maintained by**:
-  [The JAYS Maintainers](https://github.com/target/jenkins-docker-master/blob/master/MAINTAINERS)
+A Jenkins docker container. `jenkins-docker-master` was created to allow companies to run Jenkins at scale.
```

Comment:
I have very mixed feelings on this as a description. Thoughts on making this more clear?